setup.py

```diff
@@ -65,7 +65,6 @@ def read_version():
         "Environment :: Other Environment",
         "Intended Audience :: Developers",
         "Intended Audience :: Information Technology",
-        "License :: OSI Approved :: Apache Software License",
```

Comment:
Should we **replace** this with something else? Is there a new alternative to do this instead of the old License classifier?

If this is currently visible on the generated package, we don't want to just suddenly stop adding the license information.